test/libsolidity/syntaxTests/comments/multiline_unicode_direction_override_1.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.7.0;
+
+contract C {
+    function f() public pure
+    {
+        // pop 1
+        /*underflow ‬*/
```

Comment:
These characters are completely invisible when reviewing on github. What do you think about adding some additional comments to show where they are? For example:

```suggestion
        // pop 1
        /*underflow ‬*/
        //          ^
        //         PDF
```

or

```suggestion
        // pop 1
        /*underflow ‬*/

        // C++ string representation of the comment above: "/*underflow \u203c*/"
```

I had to check out the branch and open it in vim to see what's the difference between `multiline_unicode_direction_override_1.sol` and `multiline_unicode_direction_override_2.sol`.

Better test naming could help too. E.g. `multiline_unicode_direction_override_pdf_underflow.sol` and `multiline_unicode_direction_override_double_pdf_underflow.sol`.

Reply:
@mijovic Could you add some indication which character a test is using? To check, I have to open the test in an editor (or copy&paste from github).

Also, I think these tests only check PDF and RLO. At least those where I checked. Could you use each character in at least one test? Also, a test that mixes them all to make sure they all work with PDF might be a good idea.